sdcm/cluster.py

```diff
@@ -605,6 +606,12 @@ def cpu_cores(self):
         cores = int(cores) if cores else None
         return cores
 
+    @cached_property
+    def scylla_shards(self):
```

Comment:
1) convert to property
2) it should return number of shards from the server (check  how smp is added in `sdcm.cluster.BaseNode.config_setup`)

Reply:
Done

Reply:
done

Reply:
we need to check SMP too

Reply:
done